code/game/machinery/doors/airlock.dm

```diff
@@ -584,6 +589,31 @@
 	if((!(. = airlock_overlays[iconkey])))
 		. = airlock_overlays[iconkey] = mutable_appearance(icon_file, icon_state)
 
+/obj/machinery/door/airlock/proc/check_unres() //unrestricted sides. This overlay indicates which directions the player can access even without an ID
+	if(hasPower() && unres_sides)
+		if(src.unres_sides & 1)
```

Comment:
Please use direction defines in these and src is unnecessary.

Reply:
By directional defines, do you mean setting specifically to North, East, etc rather than using bitflags?

Reply:
Just mean `unres_sides & NORTH` instead of `unres_sides & 1`